### PR TITLE
Change "Openings" to "Help Wanted"

### PR DESCRIPTION
**Pull request by TheGlitch76**
We don't want to push joining a team as the primary way to contribute to Quilt. It's not necessary, and we expect people to contribute some *before* joining a team, the opposite of what parts of the website currently suggest.

This PR changes the "Openings" page to a "Help Wanted" page that is hopefully much less intimidating (it only asks for contribution, not accepting responsibility) and more reflective of how contributing to Quilt actually works.

Reviews and rewording is always welcome.

TODO before merge: 
- [ ]  Add a redirect from "openings" to "help-wanted" (I don't know how to do this)
- [ ] Re-do the section on Build Tools

### CODE REVIEW

**Review by pluiedev — CHANGES_REQUESTED** [review]
The wording looks good, though the navbar currently links to the (nonexistent) openings page; also you need to change the English strings ending in `-opening` in `/src/locales/en/common.flt` to reflect this change.

**Comment by obj-obj** [comment]
You should be able to make a redirect by doing this:

```astro
---
import Redirect from "@layouts/Redirect.astro";
---

<Redirect href="url" />
```

File: src/locales/zh/common.flt [hunk 1]
```diff
@@ -11,7 +11,7 @@ serial-comma = {$prev}、{$next}
 -developer-wiki = 开发者 Wiki
 -forum = 论坛
 -import-utility = Quilt 导入工具
--openings = 空缺职位
+-help-wanted = Help Wanted (FIXME BEFORE MERGE!!!)
```
Comment (pluiedev) [inline src/locales/zh/common.flt:14]: TBH the Chinese translation is there just as a way for me to demonstrate the language switch. I'm not planning to make a full translation so you can leave that as-is for now :p

**Review by pluiedev — APPROVED** [review]

**Comment by EnnuiL** [comment]
It appears that this PR stalled, which worries me; I consider it a prerequisite for PR #90

**Comment by gdude2002** [comment]
Looks like a bad merge was pushed to `common.flt`, this is going to need fixing pretty quickly

**Comment by Southpaw1496** [comment]
Oops. I did check and everything looks fine, but I wasn't really sure if there were any auditing steps to run.

**Comment by gdude2002** [comment]
No, you don't get me - you committed a merge conflict without resolving it.

**Pull request merged**